include/seqan3/std/bit

```diff
@@ -49,3 +53,96 @@ enum class endian
 } //namespace std
 
 #endif // __cpp_lib_endian
+
+#ifndef __cpp_lib_int_pow2
```

Comment:
I think this feature macro is not necessarily correct:

`__cpp_lib_int_pow2` includes:
* `has_single_bit`
* `bit_ceil`
* `bit_floor`
* `bit_width`

`__cpp_lib_bitops` includes:
* `rotl`
* `rotr`
* `countl_zero`
* `countr_zero`
* `countr_one`
* `popcount`


From: https://github.com/gcc-mirror/gcc/blob/releases/gcc-10.2.0/libstdc%2B%2B-v3/include/std/bit

Reply:
Thank you!